src/Writer/Extension/PodcastIndex/Feed.php

```diff
@@ -106,6 +110,189 @@ public function setPodcastIndexFunding(array $value): Feed
         return $this;
     }
 
+    /**
+     * Set feed license
+     *
+     * @param array $value [identifier: string, url: string]
```

Comment:
Could you update setters that accept an array with the expected shape please:

```suggestion
     * @param array{identifier: string, url: string} $value
```

You'll get a load of `RedundantCondition` issues from Psalm regarding the exceptions which can be baselined.